php.ini-development

```diff
@@ -371,6 +371,14 @@ zend.enable_gc = On
 ; Production Value: On
 zend.exception_ignore_args = Off
 
+; Allows setting the maximum string length in an argument of a stringified stack trace
+; to a value between 0 and 1000000.
+; This has no effect when zend.exception_ignore_args is enabled.
+; Default Value: 15
+; Development Value: 15
+; Production Value: 0
```

Comment:
This should also be in the "Quick Reference" at the top of the file, that lists ini settings that differ between dev and prod.